src/index.html

```diff
@@ -22,6 +22,343 @@
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+          class="card__photo"
+        />
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__feedback">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <article>Reviews: 5</article>
+        </div>
+        <div class="card-amount">
+          <p class="card-amount__price">Price:</p>
+          <p class="card-amount__sum">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+          class="card__photo"
+        />
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__feedback">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <article>Reviews: 5</article>
+        </div>
+        <div class="card-amount">
+          <p class="card-amount__price">Price:</p>
+          <p class="card-amount__sum">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="card-hover"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+          class="card__photo"
+        />
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__feedback">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <article>Reviews: 5</article>
+        </div>
+        <div class="card-amount">
+          <p class="card-amount__price">Price:</p>
+          <p class="card-amount__sum">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
```

Comment:
The src attribute for the image should point to the correct path of the iMac image. Ensure the path is correct and the image exists at the specified location.